src/app/screens/swap/swapConfirmation/feesBlock/index.tsx

```diff
@@ -16,20 +16,19 @@ const FeeText = styled.p((props) => ({
 }));
 
 interface FeeTextProps {
-  lpFee: number;
-  lpFeeFiatAmount?: number;
-  currency: string;
+  txFee: number;
+  txFeeFiatAmount?: number;
 }
 
-export default function FeesBlock({ lpFee, lpFeeFiatAmount, currency }: FeeTextProps) {
+export default function FeesBlock({ txFee, txFeeFiatAmount }: FeeTextProps) {
   const { t } = useTranslation('translation', { keyPrefix: 'SWAP_CONFIRM_SCREEN' });
   return (
     <Container>
       <RowContainer>
         <TitleText>{t('FEES')}</TitleText>
-        <FeeText>{`${lpFee.toFixed(6)} ${currency}`}</FeeText>
+        <FeeText>{`${txFee.toFixed(6)} STX`}</FeeText>
       </RowContainer>
-      <EstimateUSDText>{` ~ $${lpFeeFiatAmount} USD`}</EstimateUSDText>
+      <EstimateUSDText>{` ~ $${txFeeFiatAmount} USD`}</EstimateUSDText>
```

Comment:
is txFeeFiatAmount here converted to the user's currency? we need to display the correct currency instead of always USD